cg/services/order_validation_service/rules/sample/rules.py

```diff
@@ -148,3 +151,14 @@ def validate_well_position_format(
             error = WellFormatError(sample_index=sample_index)
             errors.append(error)
     return errors
+
+
+def validate_sample_container_name(
```

Comment:
In general, we decided that we prefer having the rules be a bit more descriptive since many fields have multiple validations. So could we name it `validate_container_name_required` or something similar. Probably no need to mention `sample` in the name.

Reply:
Sure, lets rename it :)

Reply:
This is updated in the new commit.